src/layouts/Layout.jsx

```diff
@@ -52,7 +62,13 @@ function Layout(props) {
           <Col md={ 4 }>
             <SideNav />
           </Col>
-          <Col className='content' md={ 8 }>
+          <Col
+            className='content'
+            id='main'
```

Comment:
I was thinking about this the other day and this looks like a good opportunity to implement it

Can you wrap `{ props.children() }` in a `<main>` tag please whilst you are here. You can then transfer themain id and role tags to it if required.